src/systems/a320_systems/src/hydraulic/mod.rs

```diff
@@ -248,6 +249,82 @@ impl A320CargoDoorFactory {
     }
 }
 
+struct A320AileronFactory {}
+impl A320AileronFactory {
+    const FLOW_CONTROL_PROPORTIONAL_GAIN: f64 = 1.5;
+    const FLOW_CONTROL_INTEGRAL_GAIN: f64 = 3.;
+    const FLOW_CONTROL_FORCE_GAIN: f64 = 200000.;
+
+    const MAX_DAMPING_CONSTANT_FOR_SLOW_DAMPING: f64 = 800000.;
+
+    fn a320_aileron_actuator(bounded_linear_length: &impl BoundedLinearLength) -> LinearActuator {
+        let randomized_damping = random_from_range(
+            Self::MAX_DAMPING_CONSTANT_FOR_SLOW_DAMPING / 10.,
+            Self::MAX_DAMPING_CONSTANT_FOR_SLOW_DAMPING,
+        );
+
+        LinearActuator::new(
+            bounded_linear_length,
+            1,
+            Length::new::<meter>(0.04),
+            Length::new::<meter>(0.),
+            VolumeRate::new::<gallon_per_second>(0.02),
+            80000.,
+            1500.,
+            5000.,
+            randomized_damping,
+            Duration::from_millis(300),
+            [1., 1., 1., 1., 1., 1.],
+            [0., 0.2, 0.21, 0.79, 0.8, 1.],
+            Self::FLOW_CONTROL_PROPORTIONAL_GAIN,
+            Self::FLOW_CONTROL_INTEGRAL_GAIN,
+            Self::FLOW_CONTROL_FORCE_GAIN,
+        )
+    }
+
+    /// Builds an aileron control surface body for A320 Neo
+    fn a320_aileron_body() -> LinearActuatedRigidBodyOnHingeAxis {
+        let size = Vector3::new(3.325, 0.16, 0.58);
+        let cg_offset = Vector3::new(0., 0., -0.5 * size[2]);
+
+        let control_arm = Vector3::new(0., -0.0525, 0.);
+        let anchor = Vector3::new(0., -0.0525, 0.33);
+
+        LinearActuatedRigidBodyOnHingeAxis::new(
+            Mass::new::<kilogram>(24.65),
+            size,
+            cg_offset,
+            control_arm,
+            anchor,
+            Angle::new::<degree>(-25.),
+            Angle::new::<degree>(50.),
+            Angle::new::<degree>(-25.),
+            1.,
+            false,
+            Vector3::new(1., 0., 0.),
+        )
+    }
+
+    /// Builds an aileron assembly consisting of the aileron physical rigid body and two hydraulic actuators connected
+    /// to it
+    fn a320_aileron_assembly() -> HydraulicLinearActuatorAssembly<2> {
+        let aileron_body = A320AileronFactory::a320_aileron_body();
+
+        let aileron_actuator_outboard = A320AileronFactory::a320_aileron_actuator(&aileron_body);
+        let aileron_actuator_inbord = A320AileronFactory::a320_aileron_actuator(&aileron_body);
+
+        HydraulicLinearActuatorAssembly::new(
+            [aileron_actuator_outboard, aileron_actuator_inbord],
+            aileron_body,
```

Comment:
```suggestion
        let aileron_actuator_outboard = A320AileronFactory::a320_aileron_actuator(&aileron_body);
        let aileron_actuator_inboard = A320AileronFactory::a320_aileron_actuator(&aileron_body);

        HydraulicLinearActuatorAssembly::new(
            [aileron_actuator_outboard, aileron_actuator_inboard],
            aileron_body,
```

Reply:
omg lol
By the way is this a proper naming ?  Could be inward / outward.... What do we say on a plane? :D